linetools/analysis/plots.py

```diff
@@ -81,7 +85,9 @@ def stack_plot(abslines, vlim=[-300,300.]*u.km/u.s, nrow=6, show=True, spec=None
         norm_sig = iline.analy['spec'].sig/co
         #
         # Plot
-        velo = iline.analy['spec'].relative_vel((1+iline.z)*iline.wrest)
+        if zref is None:
```

Comment:
Include in the test in test_plots.py

Reply:
Done.  Also, important point:  absline redshifts need to be set with AbsLine.setz() now since the vlims refactor.  A number of tests set redshifts in the old way.